src/commands/publish.ts

```diff
@@ -275,6 +275,10 @@ function getLatestMajorVersion(versions: any[]): number {
 }
 
 async function runBuild(): Promise<void> {
+    if (skipBuild) {
```

Comment:
I wonder if we can use a `useYarn` flag so we don't have to skip the build that is probably a good thing to have. Unless you think that people use more esoteric build systems that we are not even aware of yet!

Reply:
@chris-decker-volusion pointed out that there is also [PNPM](https://pnpm.js.org/) and by the time I finish typing this, there may be another one.